frameworks/digital-outcomes-and-specialists/questions/brief-responses/dayRate.yml

```diff
@@ -14,7 +14,7 @@ validations:
     message: 'You need to answer this question.'
   - name: not_money_format
     field: dayRate
-    message: "Day rate must be in numbers only, no symbols, eg 99.95"
+    message: "Day rate must be in numbers and not include symbols, for example 99.95."
```

Comment:
Do we need to say what the numbers represent - i.e. Day rate in pounds sterling?
Does the pricing question type show the unit on front of the box anyway though? In which case it should be clear.

"Day rate can only include numbers and an optional decimal point, for example 100 or 99.95." ?? Too much info?